### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by gthelohan** [carltongibson/django-filter#1512] [Q&A]
Hello, I use django-filter as a third-party app on a django project, which is running on a docker container.
I need to translate "contains" (`#: .\conf.py:29 .\conf.py:30`) in french. 
How can I make the translation and contribute ? Thank you

**Comment by carltongibson** [comment]
Hi @gthelohan — Yes, please do. I guess the message files will need updating with a `makemessages` https://docs.djangoproject.com/en/4.0/ref/django-admin/#django-admin-makemessages

**Reply by gthelohan** [reply]
Hi @carltongibson, 
I cloned the repo, did `makemessages -l fr` and `compilemessages -l fr`. How can I push to contribute ? Is there a reviewal process ? I'm not used to github public contributions. Thanks

**Reply by carltongibson** [reply]
Great. Have a look here:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

Let me know if you get stuck! 

Thanks, and welcome aboard!

**Reply by gthelohan** [reply]
I think it's ok, I made a PR :

https://github.com/carltongibson/django-filter/pull/1513 

Thanks for your help

**Reply by carltongibson** [reply]
Super. I will take a look!
